src/main/java/org/biouno/unochoice/model/ScriptlerScript.java

```diff
@@ -269,7 +269,7 @@ public JSONArray getParameters(String scriptlerScriptId) {
             if (scriptler != null) {
                 ScriptlerManagement scriptlerManagement = (ScriptlerManagement) scriptler;
                 final Script script = scriptlerManagement.getConfiguration().getScriptById(scriptlerScriptId);
-                if (script != null && script.getParameters() != null) {
+                if (script != null) {
```

Comment:
(`getParameters()` is annotated with `@NotNull`, just another change to make IDE happier)